data/add_coordinates.py

```
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import json
+import time
+import requests
+from typing import Dict, Tuple, Optional
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)